common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1232,6 +1235,34 @@
                     });
                 }
             },
+            handleTranscriptCookie: function() {
+                if ($.cookie('show_transcript') === null) {
+                    return;
+                }
+                if ($.cookie('show_transcript') !== 'false') {
+                    this.state.hideCaptions = false;
+                    // keep it going until turned off or in case of null initially change to true
+                    $.cookie('show_transcript', 'true', {
```

Comment:
Can we move this to a simple method so that we don't have to copy-paste the code at 2 places? 
```
$.cookie('show_transcript', 'true', {
                        expires: 3650,
                        path: '/'
                    });
```